packages/docs/src/stories/components/breadcrumb-item.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import { OverviewFormatter } from '../../Overview.jsx';
+
+import * as SdBreadcrumbItemStories from './breadcrumb-item.stories';
+
+export const links = {
+  'storybook-docs': '/?path=/docs/components-sd-breadcrumb-item--docs',
+  'figma-library': '',
```

Comment:
@MartaPintoTeixeira @paulovareiro29 can you both sync so that the figma branch is published before we merge this PR and we can add the links here? Same for the `sd-breadcrumb`